src/redux/products/products.types.js

```diff
@@ -2,7 +2,7 @@ export const SET_PRODUCT = 'SET_PRODUCT';
 export const GET_PRODUCT = 'GET_PRODUCT';
 export const SET_PRODUCT_LOADING = 'SET_PRODUCT_LOADING';
 export const SET_ALL_PRODUCTS = 'SET_ALL_PRODUCTS';
-export const GET_ALL_FILTERS = 'GET_ALL_PRODUCTS';
+export const GET_ALL_FILTERS = 'GET_ALL_PRODUCTS'; // before was GET_ALL_PRODUCTS
```

Comment:
Please change this string to `GET_ALL_FILTERS `

Reply:
fixed